templates/managed/index.html

```diff
@@ -1,450 +1,441 @@
+  <div class="u-fixed-width">
+    <div class="col-10">
+      <p class="p-logo-section__title u-align--center">We manage clouds for world leaders</p>
+      <div class="p-logo-section">
+        <div class="p-logo-section__items">
+          <div class="p-logo-section__item">
+            {{ image (
+            url="https://assets.ubuntu.com/v1/25de1877-Tele2.svg",
+            alt="Tele2",
+            width="253",
+            height="100",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image (
+            url="https://assets.ubuntu.com/v1/17a1b35b-telefonica_logo.svg",
+            alt="Telefonica",
+            width="262",
+            height="78",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image (
+            url="https://assets.ubuntu.com/v1/ee5d503f-rabobank-4.svg",
+            alt="Rabo bank",
+            width="2500",
+            height="453",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image (
+            url="https://assets.ubuntu.com/v1/231a1b4b-Lexis+Nexis.png",
+            alt="Lexis Nexis",
+            width="533",
+            height="300",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image (
+            url="https://assets.ubuntu.com/v1/b8e1b994-Onetrail.png",
+            alt="One trail",
+            width="350",
+            height="136",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image (
+            url="https://assets.ubuntu.com/v1/3595713f-barnes-and-noble-logo.png",
+            alt="Barnes & Noble",
+            width="180",
+            height="104",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image (
+            url="https://assets.ubuntu.com/v1/2616b34f-ACI-Universal-Payments.svg",
+            alt="ACI Universal Payments",
+            width="174",
+            height="80",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image (
+            url="https://assets.ubuntu.com/v1/cf962284-docapost.png",
+            alt="docapost",
+            width="800",
+            height="600",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+            }}
+          </div>
         </div>
+      </div>
```

Comment:
```suggestion
      </div>
      <a href="/engage/private-cloud-cost-analysis-webinar">Watch  “Hosted private cloud infrastructure: A cost analysis” webinar&nbsp;&rsaquo;</a>
```
There is this link in the doc, not sure if it is supposed to be included

Reply:
It is, but it was originally supposed to be under a different section, will ask a pm cause it looks kinda strange in the logo section